packages/kotti-ui/source/kotti-comment/KtCommentInput.vue

```diff
@@ -1,140 +1,78 @@
 <template>
-	<div :class="containerClass">
-		<div :class="wrapperClass">
-			<KtAvatar
-				v-if="!isInline"
-				class="kt-comment-input__avatar"
-				size="sm"
-				:src="userAvatar"
-			/>
-			<textarea
-				ref="textarea"
-				v-model="text"
-				class="kt-comment-input__textarea"
-				:placeholder="placeholder"
-				@blur="textFocused = false"
-				@focus="textFocused = true"
-				@input="updateHeight"
-			/>
-			<KtButton
-				:disabled="!text"
-				:label="replyButtonText"
-				type="text"
-				@click="handleSubmitClick"
-			/>
-		</div>
+	<div class="kt-comment-input">
+		<KtAvatar size="sm" :src="userAvatar" />
+		<CommentTextArea
+			v-model="_message"
+			:allowInternal="allowInternal"
```

Comment:
mostly can be v-bound

v-bind={allowInternal, autoFocus, isReply, placeholder, tabIndex }

Reply:
even the handlers technically because they are following the onEvent convention

Reply:
Skipping the change for the events for now, otherwise I'll have to either do: 
```
v-on="{
	cancel: onCancel,
	confirm: onConfirm,
	toggleInternal: onToggleInternal,
}"
```
or rename the events to: cancel, confirm, toggleInternal.
If required, I'll do this change on another PR.